app/services/teamdynamix_api.rb

```diff
@@ -97,51 +96,23 @@ def parse_response(res)
     end
   end
 
-  def valid_auth_token?(token)
-    token.match(/^[a-zA-Z0-9\.\-\_]*$/)
-  end
-
   def retire_asset_payload(asset_id)
     asset = get_asset(asset_id)
     asset.merge!(API_CONFIG[:delete].stringify_keys)
   end
 
   def create_asset_payload(host)
+    @host = host
```

Comment:
Should we be storing the host in a singleton class? We don't want to accidentally reuse it.

Reply:
This was to support configurations as, value: "created for #{@host.attribute_name}". Since we are updating it with the parameter it will be fine but we don't really have to support it and I will delete it in the next commit.